src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/DefaultGATKVariantAnnotationArgumentCollection.java

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.hellbender.cmdline.GATKPlugin;
+
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Arguments for requesting VariantContext annotations to be processed by {@link org.broadinstitute.hellbender.tools.walkers.annotator.VariantAnnotatorEngine}
+ * for tools that process variants objects.
+ */
+public class DefaultGATKVariantAnnotationArgumentCollection extends GATKAnnotationArgumentCollection {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Argument collection constructor which defines tool default annotation arguments when they are not overridden by the user
+     *
+     * @param defaultGroups               List annotation group names to be used by default
+     * @param defaultAnnotations          List of annotation class names to be used by default
+     * @param defaultAnnotationsToExclude List of annotation class names to exclude by default. These override the default annotations and annotation groups.
+     */
+    public DefaultGATKVariantAnnotationArgumentCollection(List<String> defaultGroups, List<String> defaultAnnotations, List<String> defaultAnnotationsToExclude) {
```

Comment:
This is true, I have ditched this constructor in my next branch but because this branch does not contain any of the work to connect the descriptor to argument parsing I still need this constructor on the argument collection for all the classes that need some way to use default annotations. In short, I will fix this in the next branch.